governance/processes.md

```diff
@@ -113,15 +116,51 @@ issue, but must recuse themselves from voting on the issue.
   Each voter can vote zero or more times, once per each candidate. As this is not about
   ranking but about capabilities, voters vote on a yes/neutral/no basis
   per candidate -- “would I trust this person to lead ArviZ?”.
+  - All core contributors can vote in an election.
 * Candidates are evaluated independently, each candidate having 60% or more of
-  yes votes _and_ less or equal than 20% of no votes is chosen. If the number
-  of chosen candidates is >=4 and <=10 all candidates are confirmed and the
-  election process stops here.
+  yes votes _and_ less or equal than 20% of no votes is chosen.
+  If the council resulting from all accepted candidates matches all conditions
+  described in {ref}`this section <council_description>` all candidates are
+  confirmed and the election process stops here.
 * In the event that either not enough or too many candidates were confirmed,
   candidates are ranked by interpreting yes=+1, neutral=0 and no=-1. If too
-  many candidates were confirmed, the 10 candidates with higher rank are
+  many candidates were confirmed, the 7 candidates with higher rank are
   elected. If not enough candidates were chosen, the 4 candidates with higher
-  rank are elected.
+  rank are elected. If affiliation conditions were not met, multiple potential
+  councils will be constructed with candidates whose rank is `>i` for all possible
+  values of `i`. If any of those potential councils matches the conditions,
+  the one with smaller `i` will be chosen.
```

Comment:
Tried to add the extra behaviour for when the # or candidates is in the accepted range but still conditions aren't met. Let me know what you think.